composer.json

```diff
@@ -22,7 +22,7 @@
     "prestashop/blockcategories": "dev-master",
     "prestashop/blockcms": "dev-master",
     "prestashop/blockcmsinfo": "dev-master",
-    "prestashop/blockcontact": "dev-master",
+    "prestashop/blockcontact": "dev-feat/starter-theme",
```

Comment:
maybe also add it to the repositories section so that we're not depending on packagist?

```json
"repositories": [
    {"type": "vcs", "url": "https://github.com/prestashop/blockbanner"},
    {"type": "vcs", "url": "https://github.com/prestashop/blockcart"},
    {"type": "vcs", "url": "https://github.com/prestashop/blockcontactinfos"},
    {"type": "vcs", "url": "https://github.com/prestashop/blockcurrencies"},
    {"type": "vcs", "url": "https://github.com/prestashop/blocklanguages"},
    {"type": "vcs", "url": "https://github.com/prestashop/blocktopmenu"},
    {"type": "vcs", "url": "https://github.com/prestashop/blockuserinfo"},
    {"type": "vcs", "url": "https://github.com/prestashop/blockcontact"}
  ],
```